alarms/Telegram/telegram_alarm.py

```diff
@@ -84,7 +108,7 @@ def send_alert(self, alert, info):
 				'disable_notification': 'True'
 			}
 			try_sending(log, self.connect, "Telegram (Loc)", self.client.sendLocation, locargs)
```

Comment:
Above this line add something like: 
```
If sticker_id :
 args['sticker'] = sticker_id
```
If a sticker is specified, add it to the args.